cmd/kill/killer/kill_unix.go

```diff
@@ -0,0 +1,29 @@
+// +build linux darwin dragonfly freebsd netbsd openbsd
+
+package killer
+
+import (
+	"syscall"
+)
+
+func Kill(pids []int, safe bool) (map[int]error) {
+	var errm = make(map[int]error)
+	var err error
+
+	for _, pid := range pids {
+		err = syscall.Kill(pid, syscall.SIGTERM)
+		if err != nil {
+			if safe == true {
+				goto errloop
+			}
+			err = syscall.Kill(pid, syscall.SIGKILL)
+			if err != nil {
+				goto errloop
+			}
+		}
+		continue
```

Comment:
I agree that yours is more readable indeed. Anyway let me explain my choice of logic here. When I coded that function first time I did as you exemplified, but soon the kernel coding style knocked my brain making me to remember that avoiding error logic repetition is a good practice to prevent future problems. That coding style says, when on error, handle it in the function or loop tail.
In this particular function to avoid the the `errm[pid] = err` and `continue` in each error I did prefer to control it in the loop tail.
But it's your call here. :nerd_face: